x-pack/plugins/observability_solution/apm/public/context/entity_manager_context/entity_manager_context.tsx

```diff
@@ -63,9 +63,7 @@ export function EntityManagerEnablementContextProvider({
   );
 
   const isEntityCentricExperienceViewEnabled =
-    isEntityManagerEnabled &&
-    serviceInventoryViewLocalStorageSetting === ServiceInventoryView.entity &&
-    isEntityCentricExperienceSettingEnabled;
+    isEntityManagerEnabled && isEntityCentricExperienceSettingEnabled;
```

Comment:
I was giving it a second thought, and I think we can also remove the `isEntityManagerEnabled` from this. From now on APM only cares about the value defined on the `observability:entityCentricExperience` setting. Sorry for coming back to it.

We should remove this entire line, alongside the variables this context exports that depend on this.
https://github.com/iblancof/kibana/blob/192856-apm-promote-new-inventory-plugin-in-apm/x-pack/plugins/observability_solution/apm/public/context/entity_manager_context/entity_manager_context.tsx#L55

WDYT?
cc: @kpatticha

Reply:
Okay, I will remove the `isEntityManagerEnabled` from there and only use `isEntityCentricExperienceSettingEnabled`.

Regarding the removal of all residual code, we have a [separate issue](https://github.com/elastic/kibana/issues/194114) for that, so I wouldn't include it in the scope of this PR.